fiftyone/core/collections.py

```
@@ -13675,5 +13761,16 @@ def _add_db_fields_to_schema(schema):
     schema.update(additions)
 
 
+def _merge_paths(values):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_merge_paths`

(ANN202)